src/lightning/data/streaming/functions.py

```diff
@@ -144,6 +157,7 @@ def map(
         num_downloaders: The number of downloaders per worker.
         reorder_files: By default, reorders the files by file size to distribute work equally among all workers.
             Set this to ``False`` if the order in which samples are processed should be preserved.
+        break_when_non_empty: Whether we should break if the folder isn't empty
```

Comment:
1. should it be error instead of break? Break suggests that something is going to break.
2. non is a spelling error right? It should probably be "not"
3. This is meant to reference the output folder, correct?
```suggestion
        error_when_not_empty: Whether we should break if the output folder isn't empty
```

Reply:
done !